### PR TITLE
add multi meas initiator & non linear w tests

### PR DESCRIPTION
**Pull request by lflaherty-dstl**
Altered the linear initiator to incorporate non-linear co-ordinates and changed name to simple. Added multi measurement initiator. Altered test file to test new parts.

### CODE REVIEW

**Comment by lflaherty-dstl** [comment]
NOTE: this request will not pass until inverse function request is approved as there are dependencies :)

File: stonesoup/initiator/simple.py [hunk 1]
```diff
@@ -48,11 +52,18 @@ def initiate(self, unassociated_detections, **kwargs):
...
-class LinearMeasurementInitiator(GaussianInitiator):
+class SimpleMeasurementInitiator(GaussianInitiator):
     """Initiator that maps measurement space to state space
 
+    Works for both linear and non-linear co-ordinate input
+
     This initiator utilises the :class:`~.MeasurementModel` matrix to convert
     :class:`~.Detection` state vector and model covariance into state space.
+
+    Utilises the CartesianToBearingRange inverse function to convert
```
Comment (sdhiscocks) [inline stonesoup/initiator/simple.py]: This comment should be updated, as in works for more than just the _CartesianToBearingRange_ model.

Comment (lflaherty-dstl) [reply stonesoup/initiator/simple.py]: Thanks Steve! Will do this when the branch is ready for re-testing i.e. when the inverse function is merged :)

**Comment by sdhiscocks** [comment]
> Does this just support a set of M consecutive detections or does it allow for M of N?

It can indirectly driven by the deleter, which can differ from the one in the tracker itself (recently experimented with this using 3 of 5 for initialisation, but more forgiving deleter in the main tracker).

**Pull request merged**